infrastructure/parameters/dev.parameters.json

```diff
@@ -20,6 +20,12 @@
     "publishReleaseContentCronSchedule": {
       "value": "0 30 * * * *"
     },
+    "azureDataFactoryName": {
+      "value": "s101d01datafactory"
```

Comment:
It should be possible to ditch this and instead generate this as a variable as per other resources.

Reply:
I agree but we can't do this right now as we aren't going to automate the creation of the datafactory pipeline in Azure environments until this one has been tested. I've added this comment to the description of EES-1074.

Reply:
Done